docs/source/install/kernel_install.rst

```diff
@@ -3,9 +3,13 @@
 Installing the IPython kernel
 =============================
 
-The Jupyter Notebook and other frontends automatically ensure that the IPython kernel is available.
+The Jupyter Notebook and other frontends automatically ensure that the IPython kernel is available
+for whatever version of Python you used to install the Jupyter Notebook.
```

Comment:
I'd `s/whatever/the/` here - can't really explain why, it just sounds better in my head.